hyperdbg/hprdbgctrl/code/debugger/core/symbol.cpp

```diff
@@ -385,13 +385,13 @@ SymbolConvertNameOrExprToAddress(string TextToConvert, PUINT64 Result)
     BOOLEAN IsFound            = FALSE;
     BOOLEAN HasError           = NULL;
     UINT64  Address            = NULL;
-    string  ConstTextToConvert = TextToConvert;
 
     if (!ConvertStringToUInt64(TextToConvert, &Address))
     {
         //
         // Check for symbol object names
         //
+        string ConstTextToConvert = TextToConvert;
```

Comment:
here the `ConstTextToConvert` is redundant and we can simply use the `TextToConvert` parameter!

```c
        string ConstTextToConvert = TextToConvert;
        Address = ScriptEngineConvertNameToAddressWrapper(ConstTextToConvert.c_str(), &IsFound);
```